src/lib/RangeHelper.js

```diff
@@ -386,6 +386,127 @@ define(function (require) {
+		 * @return undefined
+		 * @function
+		 * @name splitElement
+		 * @memberOf RangeHelper.prototype
+		 * @author http://stackoverflow.com/a/27498178/551625
+		 * @author brunoais
+		 */
+		base.splitElement = function (commonParent, cutElement) {
+			var grandparent;
+			for (var parent = cutElement.parentNode; commonParent !==
+					parent; parent = grandparent) {
+				var right = parent.cloneNode(false);
+
+				while (cutElement.nextSibling) {
+					right.appendChild(cutElement.nextSibling);
+				}
+
+				grandparent = parent.parentNode;
+				grandparent.insertBefore(right, parent.nextSibling);
+				grandparent.insertBefore(cutElement, right);
+			}
+		};
+
+		/**
+		 * Replace elements with provided selector inside user selection with
+		 * their children
+		 *
+		 * This method provides an easier way to have the same final aspect
+		 * for HTML as powerful as the browser's native execCommand() method
+		 * when removing HTML tag formatting.
+		 *
+		 * This method is mostly meant to be used with <b>inline</b> HTML
+		 * Elements. Having it working with block level elements is not
+		 * guaranteed.
+		 *
+		 *
+		 * @param {String} removeElementSelector Selector match what to delete
+		 * @return undefined
+		 * @function
+		 * @name splitAtSelection
+		 * @memberOf RangeHelper.prototype
+		 * @author brunoais
+		 */
+		base.splitAtSelection = function (removeElementSelector) {
+
+			var commonParent = base.getFirstBlockParent();
+
+			base.insertMarkers();
+
+			var startMarkerElement = base.getMarker(startMarker);
+			var endMarkerElement = base.getMarker(endMarker);
+			startMarkerElement.parentNode.normalize();
+
+			base.splitElement(commonParent, startMarkerElement);
+			base.splitElement(commonParent, endMarkerElement);
+
+			if (!removeElementSelector) {
+				base.removeMarkers();
+				return;
+			}
+
+			var parentOfRemoval = startMarkerElement.nextElementSibling;
+
+			while (parentOfRemoval && parentOfRemoval.id !== endMarker) {
+
+				var removalElement = ($(parentOfRemoval).
+					is(removeElementSelector) && parentOfRemoval) || null;
+
+				var removalElements = parentOfRemoval.
+					querySelectorAll(removeElementSelector);
+
+				var fragment;
+				// This has to be after querySelectorAll to allow specifying a
+				// selector for multiple HTML tags forming the removal
+				if (removalElement) {
+					fragment = document.createDocumentFragment();
+					while (removalElement.firstChild) {
+						fragment.appendChild(removalElement.firstChild);
+					}
+					removalElement.parentNode.replaceChild(fragment,
+						removalElement);
+				}
+
+				for (var i = 0; i < removalElements.length; i++) {
+					fragment = document.createDocumentFragment();
+					while (removalElements[i].firstChild) {
+						fragment.appendChild(removalElements[i].firstChild);
+					}
+					removalElements[i].parentNode.replaceChild(fragment,
+						removalElements[i]);
+				}
+
+				parentOfRemoval = parentOfRemoval.nextElementSibling;
```

Comment:
>Given a jQuery object that represents a set of DOM elements, the .next() method allows us to search through the immediately following sibling of these elements in the DOM tree and construct a new jQuery object from the matching elements.

As `next()` is only being called on one element it will just return a jQuery object containing only the next element. It also only matches elements so it will give the same result `nextElementSibling`.